stdpopsim/catalog/MusMus/species.py

```diff
@@ -69,8 +69,8 @@
 # generation in C57BL/6 laboratory mice." The provided estimate of mtDNA
 # mutation rate is derived from Goios et al. (2007). In this paper, the authors
 # say "we obtained an overall substitution rate for the mouse coding mtDNA
-# of 3.7 × 10−8 substitutions per site per yr." This value is divided by 0.75,
```

Comment:
In implementing the QC code for this I realized that we should probably be multiplying by the average generation time and not dividing. I changed the text in the documentation and also the mutation rate below.
@peterdfields - please check and confirm or correct

Reply:
I think my stoichiometry might have lapsed but if we wanted to convert from substitutions per year to substitutions per generation, and if there's 1 year / 0.75 generations,  I think dividing the reported value by 0.75 gives us substitutions per/generation?

Reply:
but it's 1 generation per 0.5 years and not 1 year per generation (0.75 is the average generation time, not generations per year).

Reply:
I think your calculation is correct here, so yes, multiplying by 0.75 seems correct. I think I somehow got mixed up in the very beginning with the numerator and denominator of generation and year and that error carried through. Sorry about that.